controllers/servers/host_definer/storage_manager/host.py

```diff
@@ -0,0 +1,42 @@
+from controllers.common.csi_logger import get_stdout_logger
```

Comment:
please replace this file with an unimplemented version of [this one](https://github.com/IBM/ibm-block-csi-driver/blob/feature/CSI-4165_host_definer_server/controllers/servers/host_definer/storage_manager/host_definer_server.py)

Reply:
done